packages/graphiql-react/src/toolbar/button.tsx

```diff
@@ -29,8 +39,12 @@ export const ToolbarButton = forwardRef<
     [onClick],
   );
 
+  const translatedLabel =
```

Comment:
This is not OK. If you use the nullish coalescing operator `??` you might not end to the default labelKey if label is empty string. Better use `||` instead or extend the expression. Also why is this `translationService?.` optional? We should expect that the service will always be there!

Reply:
There is a lint error that the service can be undefined when building the project. This is why I made it optional. I will see if I can avoid it.

Reply:
Done